toc_config.py

```diff
@@ -152,7 +153,7 @@ def parse_chapter(docname, doc, parent):
                 u'target_url': config[u'target_url'],
                 u'target_category': config[u'target_category'],
                 u'category': config[u'category'],
-                u'status': u'unlisted',
+                u'status': config[u'status'],
```

Comment:
Should use `config.get(u'status', u'unlisted')`, similar to `exercise` type exercises. This block actually is for exercisecollection, which is implemented in #23.